vespa-osgi-testrunner/src/test/java/com/yahoo/vespa/testrunner/VespaCliTestRunnerTest.java

```diff
@@ -44,26 +44,27 @@
     void testSetup() throws IOException {
         Path temp = Files.createTempDirectory("vespa-cli-test-runner-test-");
         temp.toFile().deleteOnExit();
-        VespaCliTestRunner runner = new VespaCliTestRunner(temp);
+        Path tests = Files.createDirectory(temp.resolve("tests"));
+        Path artifacts = Files.createDirectory(temp.resolve("artifacts"));
+        VespaCliTestRunner runner = new VespaCliTestRunner(artifacts, tests);
         assertFalse(runner.isSupported());
 
-        Path tests = Files.createDirectory(temp.resolve("tests"));
+        Path systemTests = Files.createDirectory(tests.resolve("system-test"));
```

Comment:
This is not used until line 59, so consider moving declaration down.

Reply:
It's the thing that makes the test "supported", so it needs to stay precisely where it is :)

Reply:
Ah, for some reason I didn't see the multiplex declaration (w/ createDirectory()).